src/main/java/duke/DeleteCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+
+public class DeleteCommand extends Command {
+
+    DeleteCommand(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Processes input given by the user to return the delete index
+     *
+     * @param input
+     */
+    Integer process(String input) throws DukeException {
```

Comment:
Can consider a more descriptive method name such as process<what> or process<into> but this is sufficient!